tools/observability/README.md

```diff
@@ -1 +1,35 @@
-[See docs here](https://docs.determined.ai)
\ No newline at end of file
+[See docs here](https://docs.determined.ai)
+
+# Metrics
```

Comment:
I know I said this was fine to put in the README but is there any reason why we shouldn't put it in the docs?

I think an advantage is the docs have the search functionality and everything will be in one place

Reply:
I can put it in the docs; a lot of the links are to other github READMEs; putting it in the README felt like a similar level of formality but I understand what you're saying about searchability and discoverability.

Reply:
If you want the algolia search to index this content (and make it discoverable in search results), convert the README to a new page, maybe "metrics.rst"